### ISSUE TITLE
Issue with "ARIA": </progress> end tag missing in code example

### ISSUE THREAD

**Issue by anandkdesign** [mdn/content#13074]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

#### What information was incorrect, unhelpful, or incomplete?

```html
<!-- this element should be close with </progress> instead of dive -->
<progress id="percent-loaded" valuenow="75"
     min="0" max="100">
</div>
```

#### Specific section or headline?
Progress code

#### What did you expect to see?
<progress id="percent-loaded" valuenow="75"
     min="0" max="100">
</progress>

#### Did you test this? If so, how?
Yes

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/index.md
* Last commit: https://github.com/mdn/content/commit/dea9592b6bcbb7127eab56218c514d52cd9b0087
* Document last modified: 2022-02-09T17:32:27.000Z

</details>

**Comment by anandkdesign** [comment]
closing tag of the progress was not correct, there should not be div element for closing tag.
